engine/src/main/java/org/pentaho/di/job/entries/deletefiles/JobEntryDeleteFiles.java

```diff
@@ -319,26 +323,29 @@ boolean processFile( String path, String wildcard, Job parentJob ) {
               BaseMessages.getString( PKG, "JobEntryDeleteFiles.TotalDeleted", String.valueOf( totalDeleted ) ) );
           }
           isDeleted = true;
-        } else {
+        }
+        else {
 
-          if ( log.isDetailed() ) {
-            logDetailed( BaseMessages.getString( PKG, "JobEntryDeleteFiles.ProcessingFile", path ) );
-          }
-          isDeleted = fileFolder.delete();
-          if ( !isDeleted ) {
-            logError( BaseMessages.getString( PKG, "JobEntryDeleteFiles.CouldNotDeleteFile", path ) );
-          } else {
-            if ( log.isBasic() ) {
-              logBasic( BaseMessages.getString( PKG, "JobEntryDeleteFiles.FileDeleted", path ) );
+            if ( log.isDetailed() ) {
```

Comment:
Please verify that the spacing here is correct; Pentaho code standards specify two spaces of indent for each level of nesting in code.

Reply:
Updated spacing for the respective line.